protocol/bin/flow.js

```diff
@@ -376,3 +381,60 @@ type CommonResponseHandler = {
   const toWrite = [typePrelude, codeGenerators.channelMapPrelude, typeDefs.join('\n\n'), incomingMap].join('\n')
   fs.writeFileSync(project.out, toWrite)
 }
+
+function decapitalize (s) {
+  return s.charAt(0).toLowerCase() + s.slice(1)
```

Comment:
There's a `camelcase` and a `titlecase` package in `npm` that does this more thoroughly. Can we use them?